.gitignore

```diff
@@ -15,3 +15,61 @@ mingw_build/
 .vscode
 cmake-build-debug/
 cmake-build-release/
+.cache
+compile_commands.json
+
+resources/ALTPALS.DAT
```

Comment:
Note that resources/ subdirectory in root is not the same directory that is discussed in the 
README. When compiling, you are supposed to do this:

```
mkdir build/
cd build/
cmake ..
make
```

This will build the openomf binary and tools and everything else into a fresh build directory. It will /also/ copy the openomf.bk to build/resources/, which is also where the assets should be put (NOT the root /resources). Building and running should never ever happen from the project root.